siliconcompiler/tools/slang/elaborate.py

```
@@ -85,61 +85,211 @@ def run(self):
+        return None, None
+
+    def __get_files(self, manager, node):
         files = set()
 
-        from queue import Queue
-        nodes = Queue(maxsize=0)
-        nodes.put(tree.root)
+        from collections import deque
+        nodes = deque([node])
 
         def proc_range(range):
```

Ruff (0.15.21)

[error] Function argument `range` is shadowing a Python builtin

(A002)